App.js

```diff
@@ -0,0 +1,79 @@
+import React from 'react';
+import { Button, Typography } from "@material-ui/core";
+import { AppBar, Toolbar, IconButton } from "@material-ui/core";
+import { Menu, MenuItem } from "@material-ui/core";
+import MenuIcon from '@material-ui/icons/Menu';
+import { makeStyles } from '@material-ui/core/styles';
+import TextField from '@material-ui/core/TextField';
+import theme from "./theme"
+import { positions } from '@material-ui/system';
+
+
+/**
+ * Has the styling solution used for Material-UI components
+ * */
+const useStyles = makeStyles({
```

Comment:
Can you use ```Styled``` instead of ```makeStyles```. This lines up with styled-components a little bit more and I think will be easier down the road.
https://material-ui.com/styles/basics/#styled-components-api